cli/src/base.ts

```diff
@@ -0,0 +1,38 @@
+import { Command } from '@oclif/core';
+import { XataApiClient } from '@xata.io/client';
+import ansiRegex from 'ansi-regex';
+import chalk from 'chalk';
+import dotenv from 'dotenv';
+import fetch from 'node-fetch';
+import table from 'text-table';
+import { readAPIKey } from './key.js';
+
+export abstract class BaseCommand extends Command {
+  locale: string | undefined = undefined;
+  timeZone: string | undefined = undefined;
```

Comment:
Can we somehow init this from system env variables?

Reply:
This is for date formatting. Passing `undefined` will use the system's default.

I put this only for snapshosts in tests, to have consistency between runs I'm setting the locale and timezone to the same values just while testing.

I'll leave a comment to explain this in the code.